### DISCUSSION TITLE
why is flutter camera  package slow

### DISCUSSION THREAD

**Discussion by GaneshTamang** [community/community#113556] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

i have just recently used flutter camera package i have noticed that the capturing image from it is somewhat laggy and slow .
Is it only me or other feels the same ?

**Comment by JuanJmf01** [comment]
I have researched the issue of slow image capture in the Flutter camera package and found the following:

- **User Reports:** There are several user reports on GitHub experiencing slow image capture with the Flutter camera package. You can check [Taking pictures is too slow on some devices](https://github.com/flutter/flutter/issues/84957).
I've also encountered a user on Stack Overflow with the same issue [Flutter camera take too long to take a picture ](https://stackoverflow.com/questions/68782788/flutter-camera-take-too-long-to-take-a-picture)

- **Adjust Camera Package Settings:** Experiment with different camera package settings, such as image resolution and file format, to see if it improves capture speed.

- **Check for Updates:** Ensure you are using the latest version of the Flutter camera package.

- **Explore Other Camera Packages:** If the slowness persists and no suitable solutions are found in the documentation or community, you could explore other camera packages available in Flutter.

- **Code Optimization:** Review your code to ensure you are using best Flutter and programming practices in general. This includes avoiding performing expensive operations on the main UI thread and optimizing your app logic to be as efficient as possible.

**Comment by Musaddiq625** [comment]
Ive tested camera package on Samsung Note 8 and IPhone 8, both didnt perform well
but it worked better on Infinix Hot 40i, it also has the android 14

so It's mostly depends on the OS compatibility

**Comment by rahulshahDEV** [comment]
You're not alone! Many Flutter developers have reported the camera package can be slow, especially with image capture.

Some developers have found better performance using alternative camera packages like [camerawesome](https://pub.dev/packages/camerawesome)

i hope this gonna solve your issue !

**Comment by RoBiN-HoOd-1** [comment]
this issue is duplicate of this ticket https://github.com/flutter/flutter/issues/84957 here you can find more info.
